CMakeLists.txt

```diff
@@ -1,16 +1,8 @@
 cmake_minimum_required(VERSION 3.10)
-cmake_policy(SET CMP0074 NEW)
 
 project(ArkScriptString)
 
-set(STRING_SOURCES
-    "${CMAKE_CURRENT_LIST_DIR}/src/String.cpp"
+add_subdirectory(src)
 
-    PARENT_SCOPE
-)
-
-set(STRING_INCLUDES
-    "${CMAKE_CURRENT_LIST_DIR}/include"
-
-    PARENT_SCOPE
-)
\ No newline at end of file
+#enable_testing ()
```

Comment:
Can be removed I think, same for the 8th line, since it's commented?

Reply:
Yeah, it was juste temporary garbage that I had, sorry for the trash, now everything should be working fine and be clean ?